lib/ColorScheme.cpp

```diff
@@ -686,15 +686,15 @@ bool ColorSchemeManager::loadColorScheme(const QString& filePath)
 QList<QString> ColorSchemeManager::listKDE3ColorSchemes()
 {
     QStringList ret;
-    foreach(const QString &scheme_dir, get_color_schemes_dirs())
+    for (const QString &scheme_dir : get_color_schemes_dirs())
```

Comment:
Actualy... there is a caveat with the range-loop ... http://www.dvratil.cz/2015/06/qt-containers-and-c11-range-based-loops/ (thanks @paulolieuthier) and to avoid the Qt deep copy.. you need to either create a `const QStringList` variable or return the `const QStringList` (or `const QStringList &`) from the `get_color_schemes_dirs()`